pylint/lint/parallel.py

```diff
@@ -36,21 +34,69 @@ def _get_new_args(message):
 
 
 def _merge_stats(stats: List[CheckerStats]):
-    merged: CheckerStats = {}
```

Comment:
This could be a class function in CheckerStats.

Reply:
I thought about this but wasn't sure what was preferred. I was not sure if a function that receives two instances of the same class and returns a third should be a class function or a "external" util. What do you prefer?

Reply:
I did not read in detail and thought it was "merge another stats into this one". An util make more sense if we return a third instance. Also as this is in parallel.py it makes sense that CheckerStats would be immutable, so you're right. Maybe we could move it in the CheckerStats namespace though ?

Reply:
Good idea. I'll add it it `checkerstats.py`